clang/test/dpct/cmake_migation/case_013/case_013_cuda_compile_cubin.cpp

```diff
@@ -0,0 +1,11 @@
+// UNSUPPORTED: system-windows
```

Comment:
Line1 could be removed, lit test should be also run pass in Window platform.

Reply:
Removed the line as suggested